### PR TITLE
Extra font info

### PR DESCRIPTION
**Pull request by MicahChambers**
Added some utilities for checking what characters are actually included in a loaded font. I personally find it frustrating to only be able to discover errors by looking at the rendered text.

### CODE REVIEW

**Comment by hugovk** [comment]
@MicahChambers Thanks for the PR.

The tests are failing for Python 3 because of `unichr`: https://travis-ci.org/python-pillow/Pillow/builds/131615297

File: PIL/ImageFont.py [hunk 1]
```diff
@@ -168,6 +168,20 @@ def font_variant(self, font=None, size=None, index=None, encoding=None):
                             index=self.index if index is None else index,
                             encoding=self.encoding if encoding is None else
                             encoding)
+    def hasglyphs(self, text):
```
Comment (hugovk) [inline PIL/ImageFont.py]: Please add a newline between these defs.

**Comment by MicahChambers** [comment]
Thanks for the review. I'll look over it tonight!

File: PIL/ImageFont.py (hunk 1 above)
Comment (MicahChambers) [inline PIL/ImageFont.py]: :+1:

**Comment by MicahChambers** [comment]
Hmm. Not sure what is causing this segfault. Reading the freetype documentation I'm thinking listing the characters might be a bad idea: https://www.freetype.org/freetype2/docs/reference/ft2-base_interface.html#FT_Get_First_Char 

Basically malformed charmaps happen and can result in 4 billion iterations....

I'm going to investigate to see if there is a way to get the all the glyphs then reverse map to the primary character mapping to each glyph.... If possible.

**Pull request closed without merging**